Debugger/Main.as

```diff
@@ -29,62 +29,46 @@ namespace VehicleDebugger
 
 	void TabViewingState()
 	{
-		auto vehicle = VehicleState::ViewingPlayerVis();
-		if (vehicle is null) {
-			UI::Text("Missing vehicle");
+		auto Vis = VehicleState::ViewingPlayerVis();
+		if (Vis is null) {
+			UI::Text("\\$F00Missing vehicle vis");
 			return;
 		}
 
-		uint entityId = VehicleState::GetEntityId(vehicle);
-		UI::LabelText("Entity ID", Text::Format("%08x", entityId));
-
-		RenderVehicleState(vehicle);
+		RenderVehicleState(Vis);
 	}
 
 #if TMNEXT || MP4
 	void TabPlayerStates()
 	{
-		auto app = GetApp();
-
-		auto pg = app.CurrentPlayground;
-		if (pg is null) {
-			UI::Text("Not currently in a playground");
-			return;
-		}
+		auto App = GetApp();
 
-		auto sceneVis = GetApp().GameScene;
-		if (sceneVis is null) {
-			UI::Text("Not currently in a scene");
+		if (App.GameScene is null) {
+			UI::Text("\\$F00Not currently in a scene");
 			return;
 		}
 
-		for (uint i = 0; i < pg.Players.Length; i++) {
+		for (uint i = 0; i < App.CurrentPlayground.Players.Length; i++) {
 #if TMNEXT
-			auto player = cast<CSmPlayer>(pg.Players[i]);
+			auto Player = cast<CSmPlayer@>(App.CurrentPlayground.Players[i]);
 #elif MP4
-			auto player = cast<CTrackManiaPlayer>(pg.Players[i]);
+			auto Player = cast<CTrackManiaPlayer@>(App.CurrentPlayground.Players[i]);
```

Comment:
`@` is not necessary in `cast<T>`. I'm also unsure why you're using `App.CurrentPlayground` instead of keeping it stored in the shorter `pg` variable.